pkg/app/app.go

```diff
@@ -194,6 +195,8 @@ func initConfiguration(engine workflow.Engine, config configuration.Configuratio
 	config.AddDefaultValue(configuration.CACHE_PATH, configuration.StandardDefaultValueFunction(dir))
 	config.AddDefaultValue(configuration.AUTHENTICATION_SUBDOMAINS, configuration.StandardDefaultValueFunction([]string{"deeproxy"}))
 	config.AddDefaultValue(configuration.MAX_THREADS, configuration.StandardDefaultValueFunction(runtime.NumCPU()))
+	// set default filesize threshold to 512MB
+	config.AddDefaultValue(configuration.IN_MEMORY_THRESHOLD_BYTES, configuration.StandardDefaultValueFunction(fmt.Sprintf("%d", 512*1024*1024)))
```

Comment:
Question: out of curiosity, why do we need `fmt.Sprintf()`? can't we us an integer directly?